cub/cub/detail/nvtx.cuh

```diff
@@ -47,14 +47,15 @@
 // * NVTX is not explicitly disabled (via CCCL_DISABLE_NVTX or NVTX_DISABLE)
 // * C++14 is availabl for cuda::std::optional
 // * NVTX3 uses module as an identifier, which trips up NVHPC
-#if __has_include(<nvtx3/nvToolsExt.h> ) && !defined(CCCL_DISABLE_NVTX) && !defined(NVTX_DISABLE) \
-&& _CCCL_STD_VER >= 2014 &&(!defined(_CCCL_COMPILER_NVHPC) || _CCCL_STD_VER <= 2017)
+#if _CCCL_HAS_INCLUDE(<nvtx3 / nvToolsExt.h>) && !defined(CCCL_DISABLE_NVTX) && !defined(NVTX_DISABLE) \
```

Comment:
I am unsure whether that here is a but or whether __has_include still works
```suggestion
#if _CCCL_HAS_INCLUDE(<nvtx3/nvToolsExt.h>) && !defined(CCCL_DISABLE_NVTX) && !defined(NVTX_DISABLE) \
```

Reply:
Oh, it must have been done by `clang-format`. I will try to fix it.

Reply:
Fixed.